### ISSUE TITLE
need "how to install bikeshed in one's local webauthn repo clone" instructions

### ISSUE THREAD

**Issue by equalsJeffH** [w3c/webauthn#1477]
I was attempting to run the `./update-bikeshed-cache.sh` on my local webauthn repo clone (following the directions here: https://github.com/w3c/webauthn#updating-copies-of-bikeshed-data-files-stored-in-this-repo) and this is what I got:
```
$ ./update-bikeshed-cache.sh \
  && git add .spec-data .bikeshed-include \
  && git commit -m "Bikeshed spec data update" .
Precondition failure: expecting a bikeshed installation in ./bikeshed/
```
So that is saying that it is expecting a bikeshed install to be in `<my local path>/webauthn/bikeshed`.

In looking at https://tabatkins.github.io/bikeshed/#installing it is not clear to me how to place a "bikeshed installation" in that directory, and I'm afraid of messing up my present local bikeshed install (which is sort of a baroque mess because of the mess of python installations and environments I seem to have...).

Anyone have clean & concise instructions for how to place a "bikeshed installation" in `<my local path>/webauthn/bikeshed` **_without messing up anything_** on the local machine?

**Comment by emlun** [comment]
I use the [Docker image](https://github.com/w3c/webauthn/tree/master/docker/bikeshed) to build the spec. It looks like you could just build that image, then extract the `/user/local/lib/python3.8/site-packages/bikeshed/boilerplate/webauthn/` and `.../bikeshed/spec-data/` directory contents from it. I just pushed [this script](https://github.com/w3c/webauthn/blob/update-bikeshed-cache-docker/update-bikeshed-cache-docker.sh) on the `update-bikeshed-cache-docker` branch, try if that works.

**Comment by jcayzac** [comment]
I struggled with the instructions as well. I'd suggest moving to Docker completely and stop relying on installing Python and packages on the host (just use volume mounting inside Docker). I can have a look at this later in the week.

**Comment by jcayzac** [comment]
By the way, I was curious when I started to fiddle with the bikeshed instructions mentioned above: is the repo based on some w3c template?

**Comment by equalsJeffH** [comment]
> is the repo based on some w3c template?

do not know, it may be, that's a question for @wseltzer I think.

**Comment by wseltzer** [comment]
> is the repo based on some w3c template?

New W3C repos are generally created with the https://github.com/w3c/ash-nazg tool , but this one pre-dates that practice.

**Comment by equalsJeffH** [comment]
on 5-May-2021 call:
the [present instructions in the README.md file for installing bikeshed in this repo](https://github.com/w3c/webauthn#updating-copies-of-bikeshed-data-files-stored-in-this-repo), are incorrect and ought to be brought up-to-date, along with anything else that's similarly out-of-date in the README.md.  @wseltzer is going to see if @sideshowbarker may be able to help us out with this.

**Comment by sideshowbarker** [comment]
With #1625 merged, I’ve opened https://github.com/w3c/webauthn/pull/1628 to update the README.md

**Comment by equalsJeffH** [comment]
fixed by #1628
